### PR TITLE
LPS-136183

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-142027

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 1]
```diff
@@ -72,148 +72,564 @@ definition {
...
 	@description = "This is a test for LPS-136183. Can use a Masterclass Sign In fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassSignIn {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-136183.
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:462]: Move before navigation

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 2]
```diff
@@ -72,148 +72,564 @@ definition {
...
 	@description = "This is a test for LPS-136183. Can use a Masterclass Subscribe Form fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassSubscribeForm {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-136183.
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:491]: Move before navigation

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 3]
```diff
@@ -72,148 +72,564 @@ definition {
...
 
 	@description = "This is a test for LPS-136183. Can use a Masterclass Tabs fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassTabs {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
+
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:520]: Move before navigation

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 4]
```diff
@@ -72,148 +72,564 @@ definition {
...
 	@description = "This is a test for LPS-136183. Can use a Masterclass Text Block fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassTextBlock {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-136183.
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:549]: Move before navigation

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 5]
```diff
@@ -72,148 +72,564 @@ definition {
...
 	@description = "This is a test for LPS-136183. Can use a Masterclass Video fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassVideo {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-136183.
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:578]: Move before navigation

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 6]
```diff
@@ -72,148 +72,564 @@ definition {
...
 
 	@description = "This is a test for LPS-136183. Can use a Masterclass Video Player fragment"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanUseMasterclassVideoPlayer {
+		task ("Go to Masterclass Home Page") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
+
+		task ("Create a new Page by Json") {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:613]: Move before navigation

**Comment by catarineaguiar** [comment]
See #https://github.com/david-gutierrez-mesa/liferay-portal/pull/218

**Comment by catarineaguiar** [comment]
ci:close

**Pull request closed without merging**